src/store/modules/permissions.js

```diff
@@ -78,6 +78,7 @@ export default {
         'connection.open': 'address',
         'address.subscribe': 'address',
         'message.sign': 'messageSign',
+        'address.get': 'addressList',
```

Comment:
Maybe combine it with `address.subscribe`? It adapt allowed to cal `address.get` without confirmation then it can subscribe by polling.

Reply:
`askAddresses` is the `METHOD.address` request and it is `address.get`

```
  /**
   * Ask addresses from wallet
   * @returns Addresses from wallet
   */
  async askAddresses(): Promise<Encoded.AccountAddress[]> {
    this._ensureAccountAccess();
    return this.rpcClient.request(METHODS.address, undefined);
  }
  ```

Reply:
Also you can implement `address.get` to return a single address, I'm not sure that it is defined by protocol.

Reply:
I see, @davidyuk wouldn't it conflict with the all-account address list then?

> Also you can implement address.get to return a single address
I don't see a callback method like `onAskAccounts` for single address requests. Maybe something to consider for the Aepp communication improvement epic.